jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.py

```diff
@@ -137,20 +137,29 @@ def _write_moment_to_fits(self, overwrite=False, *args):
         if self.moment is None or not self.filename:  # pragma: no cover
             return
 
-        path = Path(self.filename).resolve()
-        if path.exists():
+        # Make sure file does not end up in weird places in standalone mode.
+        path = os.path.dirname(self.filename)
+        if path and not os.path.exists(path):
+            raise ValueError(f"Invalid path={path}")
+        elif (not path or path.startswith("..")) and os.environ.get("JDAVIZ_START_DIR", ""):  # noqa: E501 # pragma: no cover
+            filename = Path(os.environ["JDAVIZ_START_DIR"]) / self.filename
+        else:
+            filename = Path(self.filename).resolve()
+
+        if filename.exists():
```

Comment:
I'd REALLY like to architect a pathlib-only solution here and avoid using `os.path`. I think we could make this codeblock much more elegant with a combination of `path.exists()`, `path.parent` and `path.is_dir()`, `path.relative_to()`, and `path.is_absolute()`

I'm generally a stickler for path handling and I admit I was tempted to leave a "changes requested" review here, but I'll bite my tongue and allow myself to get overruled here since we're at the end of the sprint